test/e2e/support/utils.js

```diff
@@ -33,6 +33,24 @@ export const META_KEY = process.platform === 'darwin' ? 'Meta' : 'Control';
  */
 export const ACCESS_MODIFIER_KEYS = process.platform === 'darwin' ? [ 'Control', 'Alt' ] : [ 'Shift', 'Alt' ];
 
+/**
+ * Platform-specific modifier for the primary+alt key chord.
+ *
+ * @see pressWithModifier
+ *
+ * @type {string}
+ */
+export const PRIMARY_ALT_MODIFIER_KEYS = process.platform === 'darwin' ? [ 'Alt', 'Meta' ] : [ 'Control', 'Alt' ];
```

Comment:
Do we need this? Or is it sufficient to do `[ 'Alt', META_KEY ]` ? Same applies for the constant defined below.

Reply:
Removed.